serverlessworkflow/modules/ROOT/pages/cloud/build-workflow-image-with-quarkus-cli.adoc

```diff
@@ -1 +1,173 @@
-= Building Workflow Images locally with Quarkus CLI
\ No newline at end of file
+Building Workflow Images locally with Quarkus CLI
+=================================================
+Author Name <fspolti@redhat.com>
```

Comment:
I think we can remove this. See: https://github.com/kiegroup/droolsjbpm-build-bootstrap#faq